File: nxfit_sync_sdk/android/src/main/kotlin/com/neoex/nxfit/sync_sdk/LocalIntegration.kt
package com.neoex.nxfit.sync_sdk

import kotlinx.serialization.Serializable

@Serializable
data class LocalIntegration(
    var identifier: String,
    var isConnected: Boolean,
    var availability: String
)

@Serializable
data class LocalIntegrationList(
    val integrations: List<LocalIntegration>
)

File: nxfit_sync_sdk/ios/nxfit_sync_sdk/Sources/nxfit_sync_sdk/Models/LocalIntegration.swift
import Foundation
import NXFitModels

internal struct LocalIntegration : Encodable {
    let identifier: String
    let isConnected: Bool
    let availability: IntegrationAvailability
}

internal struct LocalIntegrationList : Encodable {
    let integrations: [LocalIntegration]
}

internal enum IntegrationAvailability : String, Encodable {
    case available = "available"
}
